ember-cli-build.js

```diff
@@ -40,7 +40,9 @@ module.exports = function(defaults) {
             [useCdn ? 'jquery.js' : '']: false,
         },
         'ember-bootstrap': {
-            importBootstrapCSS: false
+            importBootstrapCSS: false,
```

Comment:
because of line 4 in [ routes/provider.js](https://github.com/CenterForOpenScience/ember-osf-reviews/pull/2/files/1908a8109788fb5618db924e1d2e61798c2aeb47#diff-b4af49876da16fe17ebc9b388cd01841)
``
const providers = config.Reviews.providers.slice(1);
``